src/Merger/merger.go

```diff
@@ -44,6 +46,35 @@ func readCsvContent(filepath string) ([]byte, error) {
 	return fileBytes, nil
 }
 
+func MergeFromFolder(folderPath string) error {
+	_, err := os.Stat(folderPath)
+	if os.IsNotExist(err) {
+		return errors.New("Folder does not exist or could not be found. \nError: " + err.Error())
+	} else if err != nil {
+		return err
+	}
+
```

Comment:
Avoid "else" statements at all costs😉
https://medium.com/@matryer/line-of-sight-in-code-186dd7cdea88